quickstart/database/seeders/Category.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class Category extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('category')->delete();
+        DB::table('category')->insert([
+            ['id'=>1,'name'=>'Nam','parent'=>0],
```

Comment:
```suggestion
            ['id'=>1, 'name'=>'Nam', 'parent'=>0],
```
tương tự các chỗ khác nhé